apps/web/src/app/karriere/page.tsx

```diff
@@ -119,41 +123,52 @@ const CompanyAdListItem: FC<CompanyAdListItemProps> = ({ jobListing }: CompanyAd
   return (
     <Link
       href={`/karriere/${jobListing.id}`}
-      className="border-gray-200 flex h-56 items-center justify-between rounded-lg border px-6 py-2"
+      className={`border ${jobListing.featured ? "border-orange-100" : "border-gray-200"} flex h-56 items-center justify-between rounded-lg px-6 py-2 ${jobListing.featured && "bg-orange-50 dark:bg-neutral-800"}`}
```

Comment:
Could use `cn` here to make it easier to read
```suggestion
      className={cn(
        "border flex h-56 items-center justify-between rounded-lg px-6 py-2",
        jobListing.featured ? "border-orange-100 bg-orange-50 dark:bg-neutral-800" : "border-gray-200"
      )}
```